server/authorization/router/auth.router.js

```diff
@@ -12,29 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import express from 'express';
+import Application from '../../base/application.js';
 import * as auth from '../controller/auth_controller';
 
-const router = express.router();
-
-router.post('/uuid', (request, response) => {
-  auth.generateUuid().then((authInfo) => {
-    response.send(authInfo);
-  }, (error) => {
-    response.sendStatus(400);
-  });
-});
+@Application.route('/uuid')
```

Comment:
Please add a jsdoc comments for rest API.
  - example: https://github.com/romandev/absolute/pull/344/files